bin/permadelete_crash_data.py

```diff
@@ -73,6 +73,11 @@ def cmd_permadelete(ctx, crashidsfile):
 
     crashstorage = build_instance_from_settings(settings.STORAGE)
     es_crashstorage = build_instance_from_settings(settings.ES_STORAGE)
+    legacy_es_crashstorage = None
+    if settings.ELASTICSEARCH_MODE == "PREFER_NEW":
+        legacy_es_crashstorage = build_instance_from_settings(
+            settings.LEGACY_ES_STORAGE
+        )
```

Comment:
from the docs for the setting:

> When set to LEGACY_ONLY the processor and webapp will only use legacy es.
> When set to PREFER_NEW the processor will write to both and the webapp will only use new.

for LEGACY_ONLY we only write to one es cluster and only need to delete from one cluster, and ES_STORAGE == LEGACY_ES_STORAGE, so it can continue using ES_STORAGE.

for PREFER_NEW we are writing to both ES_STORAGE and LEGACY_ES_STORAGE in the processor, and as such permadelete needs to delete from both.

eventually we will remove ELASTICSEARCH_MODE, and only ES_STORAGE will exist, at which time this script only needs to have these if conditions removed.